src/v2/view-builder/views/ov/ChallengeAuthenticatorDataOktaVerifyPushOnlyFormView.js

```diff
@@ -0,0 +1,41 @@
+import { createButton, loc } from 'okta';
+import { BaseForm } from '../../internals';
+
+const Body = BaseForm.extend(Object.assign({
+  className: 'okta-verify-push-challenge',
+  noButtonBar: true,
```

Comment:
Pradeep and I spoke about this. He had also mentioned it in slack thread. If we go with default button, positioning checkbox below the button in `postRender` doesn't work. That is because the default button is added via `save()` function which is called after `postRender()`.

Reply:
Interesting. What we have is good, but if we ever change the classnames or styles of the primary button in a form, we might not get those changes in this view. I see we are adding the right classnames, so it should be ok, but maybe we can look into it in this PR or a future one